render_bundle/tox.ini

```
@@ -0,0 +1,49 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+[tox]
+skipsdist=True
+skip_missing_interpreters = True
+envlist = lint, unit, static
```

I've started adding the formatting to the `tox` command and it's pretty useful for avoiding lint errors. Feel free to resolve tho.
```suggestion
envlist = fmt, lint, unit, static
```